lib/activerecord/all/activerecord.rbi

```diff
@@ -701,34 +741,34 @@ module ActiveRecord::Persistence::ClassMethods
   sig do
     params(
       attributes: T::Array[T::Hash[T.any(Symbol, String), T.untyped]],
-      returning: T.nilable(T.any(FalseClass, T::Array[T.any(Symbol, String)]))
```

Comment:
The docs show that this has a `nil` default argument, why remove that information from the signature?